code/modules/clothing/shoes/miscellaneous.dm

```diff
@@ -198,7 +198,7 @@
 	jumping = TRUE
 	playsound(src.loc, 'sound/effects/stealthoff.ogg', 50, 1, 1)
 	usr.visible_message("<span class='warning'>[usr] dashes foward into the air!</span>")
-	usr.throw_at(target, jumpdistance, 1, spin=0, diagonals_first = 1, callback = CALLBACK(src, .proc/hop_end))
+	usr.throw_at(target, jumpdistance, 3, spin=0, diagonals_first = 1, callback = CALLBACK(src, .proc/hop_end))
```

Comment:
uh
Higher number the faster the speed, right?
It's how things with a certain speed embed, right?